tailwind.config.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
   content: [
     'src/pages/**/*.{js,jsx}',
     'src/components/**/*.{js,jsx}',
-    'src/layouts/**/*.{js,jsx}'
+    'src/layouts/**/*.{js,jsx}',
```

Comment:
```suggestion
    'src/**/*.{jsx}',
```
Này cho đơn giản luôn em nha. Xác định rõ là chỉ có mỗi file jsx sử dụng tailwind thôi, mấy file khác bỏ qua